backend.native/tests/runtime/basic/simd.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license
+ * that can be found in the LICENSE file.
+ */
+
+package runtime.basic.simd
+
+import kotlin.test.*
+
+
+@Test fun runTest() {
+    testSetGet()
```

Comment:
It makes sense to add `@Test` to all these functions instead of calling them directly.

Reply:
Following the dominating style of other tests in the folder(s) in this case. Refactoring (modernize) the whole test suite "all at once" may be suggested.

Reply:
The dominating style here is to have `main` function.
It doesn't make much sense to have `@Test` annotation but only on a single function.